public/app/core/utils/explore.ts

```diff
@@ -363,10 +363,14 @@ export function getIntervals(range: TimeRange, lowLimit?: string, resolution?: n
 }
 
 export const copyStringToClipboard = (string: string) => {
-  const el = document.createElement('textarea');
-  el.value = string;
-  document.body.appendChild(el);
-  el.select();
-  document.execCommand('copy');
-  document.body.removeChild(el);
+  if (navigator.clipboard && window.isSecureContext) {
```

Comment:
I ran into an odd issue with the code here in a menu. It would copy once, but not again without a page refresh. I opted to something more similar to what we have elsewhere

https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.tsx#L92

The bug where it only copies once from the menu will still be there, but only for users with browsers that don't support navigator.clipboard. Since elsewhere in the codebase we use navigator.clipboard without any fallback, this seems to be a small amount of users and I am not concerned about it.

Reply:
I've tested this in Chrome and Firefox and it works as expected